configs/cms-xcache/config.d/90-cms-xcache-disks.cfg

```diff
@@ -0,0 +1,17 @@
+#
+# Configure disks for cache servers of the CMS data federation
+#
+# ************************************************************
+# * NOTE: Changes to this file will be PRESERVED on upgrades *
+# ************************************************************
+#
+# This file is part of the CMS XCache Daemon
+
+# Specify the location of filesystems to hold data files. Can be
+# specified more than once with the following format:
+#
+# oss.space data <ABSOLUTE PATH>
+#
+# For example:
+#
+# oss.space data /data1/xcache
```

Comment:
I would add some more commented lines like /data1 /data2 to make sure you can have more.

Reply:
Done